lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -121,30 +121,6 @@
   end
 end
 
-desc 'Build puppet module package'
-task :build do
-  Rake::Task['build:pdk'].invoke
-end
-
-namespace :build do
-  desc 'Build Puppet module with PDK'
-  task :pdk do
-    require 'pdk/util'
-    require 'pdk/module/build'
-
-    path = PDK::Module::Build.invoke(force: true, 'target-dir': File.join(Dir.pwd, 'pkg'))
-    puts "Module built: #{path}"
-  rescue LoadError
-    _ = `pdk --version`
-    unless $CHILD_STATUS.success?
-      warn 'Unable to build module. Please install PDK or add the `pdk` gem to your Gemfile.'
-      abort
-    end
-
-    system('pdk build --force')
-  end
-end
-
 desc 'Clean a built module package'
```

Comment:
Does this still belong here? If the build part is removed. There's already a separate `spec_clean` that removes the fixtures

Reply:
No, you're right. Nice spot, thanks!